### ISSUE TITLE
Spanish-style punctuation

### ISSUE THREAD

**Issue by iandoug** [HughP/tcf-james-bigramed#12]
Does tcf also use Spanish-style matching ¡ ! and ¿ ?  ?

**Comment by HughP** [comment]
No. Not that I can tell. I think they have a question particle like 'ma' in
Mandarin Chinese. they do use << and >> for quotes like French.

**Comment by iandoug** [comment]
The corpus has two cases of inverted exclamation mark:

¡ U+00A1 2  Inverted Exclamation Mark

But I didn't see any inverted question marks, hence the question.

**Comment by HughP** [comment]
Which file in the corpus?

**Comment by iandoug** [comment]
mephaa3-unicode.txt

Line 89:
5 Xúꞌkhue̱n máꞌ jaꞌnii rí ra̱ju̱u̱n xa̱bo̱ mangaa, mbá xuwi lájwíin ñajuun ne̱, jamí phú mba̱a̱ rí na̱ꞌngo̱o̱ ne̱ naꞌne. ¡Ra̱ꞌkhá tháán mba̱a̱ júba̱ ikha eꞌne mbá lájwíin ri̱ꞌyu̱u̱ agu rí nakhati̱yo̱o̱ꞌ!

Line 115:
4 ¡Ra̱ꞌkhá xa̱bo̱ tsí nandúún juyáá i̱ndo̱ó Ana̱ꞌlóꞌ ñajwanlaꞌ! Á tsíya̱álaꞌ rí xa̱bo̱ tsí nandoo guéño jaꞌyoo rí ríga̱ ná numbaa, tsíyoo rí ma̱mbáxu̱u̱ꞌ ga̱jmáa̱ Ana̱ꞌlóꞌ rá dxe̱ꞌ. Ikhaa jngó, asndo tsáa máꞌ tsí nandoo guéño jaꞌyoo rí ríga̱ ná numbaa, tsímbáxu̱u̱ꞌ ga̱jmáa̱ Ana̱ꞌlóꞌ.
